zstd/decoder.go

```diff
@@ -289,9 +289,15 @@ func (d *Decoder) DecodeAll(input, dst []byte) ([]byte, error) {
 	}
 
 	// Allocation here:
```

Comment:
And this commend must be changed to something like: `Preventing allocation here`